backend/shared/src/websockets/server.ts

```diff
@@ -108,6 +109,10 @@ export function broadcastMulti(topics: string[], data: BroadcastPayload) {
   }
 }
 
+export function broadcastCommentUpdate(commentId: string, data: CommentUpdatePayload) {
+  return broadcast(`comment:${commentId}`, data)
```

Comment:
Please follow a similar event name scheme to updated answers: `contract/${contractId}/updated-comment`